packages/web-api/src/WebClient.spec.js

```diff
@@ -856,9 +868,35 @@ describe('WebClient', function () {
           done();
         });
     });
-    // TODO: when parsing the retry header fails
   });
 
+  it('should throw an error if the response has no retry info', function (done) {
+      const scope = nock('https://slack.com')
+        .post(/api/)
+        .reply(429, {}, { 'retry-after': undefined });
+      const client = new WebClient(token);
+      client.apiCall('method')
+        .catch((err) => {
+          assert(err.message.match(/Retry header did not contain a valid timeout/i) !== null);
```

Comment:
🤔 In case this message ever changes, I think it's good to replace this with something that [just makes sure `err` is an `Error`](https://github.com/slackapi/node-slack-sdk/blob/7247d387de3645415eac83a41d8ec0116e3730f1/packages/web-api/src/WebClient.spec.js#L164). Ditto on the other new test case below this one 👇.

Reply:
That makes sense to me, thanks for including a link to that other example.